joomla/joomla-cms
# Articles category module new filter option (modified later than create)

Pull Request for Issue # .

### Summary of Changes

New additional filter option that can hide articles modified later than provided value of minutes after create date/time.
This is useful for articles list with Date Range Field set to Modified Date - atm we can see all modified articles even when modified=created. My PR will allow to hide articles modified within i.e. 20 mins from creating date/time.
I.E we have two lists on a web page - recently created articles and recently modified. Without my code recently modified will show all articles even if modified value is null (newly created). There are times that users create article and after posting them they make a simple modification (i.e. spelling) and we don't want to list those articles as 'modified'. 

### Testing Instructions

Add new module of type Articles-Category. Set Modified Later Than value to 0 (default), Date Range Field to "modified date".

Create a few articles, after a while (a few mins. i.e. 5) modify some of them. 
Go to web page - all articles should be visible.
Modify Set Modified Later value to 4 (mins). 
Go to web page - all articles that was modified within those 4 mins after creation date/time should be invisible, only those modified > 4 min should be on list.

### Expected result

Check Testing Instructions

### Documentation Changes Required

Description of additional filter options.
